src/PolyhedralGeometry/PolyhedralFan/properties.jl

```diff
@@ -226,6 +257,23 @@ julia> n_maximal_cones(PF)
 """
 n_maximal_cones(PF::_FanLikeType) = pm_object(PF).N_MAXIMAL_CONES::Int
 
+@doc raw"""
+    n_cones(PF::PolyhedralFan)
+
+Return the number of cones of `PF`.
+
+# Examples
+The cones given in this construction are non-redundant. There are six
+cones in this fan.
+```jldoctest
+julia> PF = PolyhedralFan([1 0; 0 1; -1 -1], IncidenceMatrix([[1, 2], [3]]));
+
+julia> n_cones(PF)
+6
+```
+"""
+n_cones(PF::_FanLikeType) where T<:scalar_types = nrows(cones(PF))
```

Comment:
```suggestion
n_cones(PF::_FanLikeType) = nrows(cones(PF))
```
Same here.

Reply:
I have taken care of this - see previous suggestion.